install-wings.sh

```diff
@@ -527,6 +581,38 @@ function main {
     if [[ "$CONFIRM_FIREWALL_CMD" =~ [Yy] ]]; then
       CONFIGURE_FIREWALL_CMD=true
     fi
+
+    ask_letsencrypt
+  fi
+
+  if [ "$CONFIGURE_LETSENCRYPT" == true ]; then
+    while [ -z "$FQDN" ]; do
+        echo -n "* Set the FQDN to use for Let's Encrypt (panel.example.com): "
```

Comment:
```suggestion
        echo -n "* Set the FQDN to use for Let's Encrypt (node.example.com): "
```

We should change this because people may be confused that it is panel and that they are doing a bad thing.